src/generators/javascript/templates/index.hbs

```diff
@@ -12,6 +12,7 @@
  * You can install it with: `npm install --save-dev ajv`.
```

Comment:
Add installation step for ajv-draft-04 as well

Reply:
Yeah, will add that